keyboards/handwired/swiftrax/cowfish/rules.mk

```diff
@@ -0,0 +1,31 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = qmk-dfu
```

Comment:
Is it running QMK DFU for sure? Or is it stock Atmel DFU?

Reply:
stock amtel dfu. Should i change it to amtel?